src/main/java/com/aws/greengrass/provisioning/ProvisionConfiguration.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.provisioning;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class ProvisionConfiguration {
```

Comment:
provide builders for these?

Reply:
I was trying to avoid them unless needed. But its probably going to be useful for customers.